app/views/support_interface/support_users/new.html.erb

```diff
@@ -8,6 +8,15 @@
 
       <h1 class="govuk-heading-l">Add support user</h1>
 
+      <%# TODO: Use a component for warning text %>
+      <div class="govuk-warning-text">
+        <span class="govuk-warning-text__icon" aria-hidden="true">!</span>
+        <strong class="govuk-warning-text__text">
+          <span class="govuk-warning-text__assistive">Warning</span>
+          Only give access to the support console to approved users.
+        </strong>
+      </div>
+
```

Comment:
Sure would be nice if I could use the component from `govuk-components`… cough @duncanjbrown cough.

Reply:
Absolutely no objection to you adding it in this PR!

Reply:
Addressed in #4347